### PR TITLE
Add OpenGL ES jobs to ci.yml

### PR DESCRIPTION
**Pull request by binary1248**
Title.

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
What extra coverage do these CI jobs give us?

**Comment by binary1248** [comment]
All the OpenGL ES code paths.

**Review by eXpl0it3r — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,6 +46,12 @@ jobs:
         - platform: { name: Windows VS2022, os: windows-2022 }
           config: { name: Unity, flags: -DSFML_USE_MESA3D=TRUE -DBUILD_SHARED_LIBS=TRUE -DCMAKE_UNITY_BUILD=ON }
           type: { name: Debug, flags: -DCMAKE_BUILD_TYPE=Debug -DSFML_ENABLE_COVERAGE=TRUE }
+        - platform: { name: Windows VS2022, os: windows-2022 }
+          config: { name: OpenGL ES, flags: -DSFML_USE_MESA3D=TRUE -DBUILD_SHARED_LIBS=TRUE -DSFML_OPENGL_ES=ON }
```
Comment (ChrisThrasher) [inline .github/workflows/ci.yml:50]: You can omit `-DBUILD_SHARED_LIBS=TRUE` since they're on by default. Are shared libs truly a requirement anyways?

**Review by ChrisThrasher — APPROVED** [review]
Just one non-blocking comment.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (binary1248) [reply .github/workflows/ci.yml:50]: I just took this from the existing configurations. It wouldn't make sense to declare it sometimes and sometimes not, either we are explicit or we aren't.

**Pull request merged**